src/context/auth-context.tsx

```diff
@@ -86,18 +92,18 @@ export const AuthDataProvider = ({ children }: AuthDataProviderProps) => {
 
     const logout = useCallback(() => {
         Cookies.remove('authToken');
-        Cookies.remove('accountsList');
+        Cookies.remove('tradingAccounts');
```

Comment:
just curious, why only `tradingAccounts`, there's also wallet accounts right?

Reply:
i can change it back to accountsList